apps/utils.py

```diff
@@ -80,3 +80,15 @@ class Labels:
         PENDING = 'PENDING'
         COMPLETE = 'COMPLETE'
         FAILED = 'FAILED'
+
+
+def get_organization_name(request):
+    if request.user.is_authenticated():
+        return request.user.token.payload.get('organization_name')
```

Comment:
The simplejwt Token class implements __getitem__ so you don't need `.payload` here.

Reply:
I also think it's a nice practice to explicitly set the default value of None here.